src/ui/page/share/dataset/steps/summary/summary.tsx

```diff
@@ -58,10 +62,23 @@ export const Summary: FC = () => {
   const getFieldValueForNumeric = (item: NumericFieldType): number | undefined =>
     pipe(item.value, O.toUndefined)
 
+  const getDateRangeFieldValue = (item: DateStringRangeField): DateStringRange | undefined =>
+    pipe(item.value, O.toUndefined)
+
   type SummaryLeftItemProps = {
     item: FormItem
   }
 
+  const localizedDateFormatter = useMemo(
+    () => new Intl.DateTimeFormat(locale, { year: 'numeric', month: '2-digit', day: '2-digit' }),
+    [locale]
+  )
+
+  const formatISODate = useCallback(
+    (isoDateString: string) => localizedDateFormatter.format(new Date(isoDateString)),
+    [localizedDateFormatter]
+  )
```

Comment:
These functions should be part of the `date.ts` file under utility folder and please add unitary tests.

Reply:
Function has been added as utilities in [b24befa](https://github.com/okp4/dataverse-portal/pull/469/commits/b24befa5283f996c03fd5706cf65aa8458c63587), and tested in [5f001c6](https://github.com/okp4/dataverse-portal/pull/469/commits/5f001c6f45a93e9954d81999c663e58fc0259f1b). Thanks!